packages/patternfly-4/content/design-guidelines/usage-and-behavior/wizard/design.md

```diff
@@ -2,33 +2,45 @@
+
+The modal wizard lives in a modal dialog. The width and height of the modal is variable and should be sized to comfortably fit the contents of the wizard at typical desktop screen resolutions. In most cases, the modal wizard is recommended as it will keep users focused on the task at hand. Users must either complete all of the steps of the wizard or cancel before navigating elsewhere within an application.
+
+### In-page wizards
+![in page wizard](./img/wizard-in-page.png)
+
+Wizards can also be added to the content area of a page. This will allow greater flexibility in navigating to other locations in your application while within a wizard flow, but it also places greater responsibility on the system to manage state.
 
 ## Elements of a wizard
-The wizard lives in a modal dialog. The width of the modal is variable and should be sized to comfortably fit the contents of the wizard at typical desktop screen resolutions.
+Except where noted, modal and in-page wizards have the same elements and behaviors. For the sake of simplicity, all screens here show the modal wizard.
 
 ### Simple wizard
 
 ![simple-wizard](./img/standard-wizard.png)
 
-1. **Header:** Wizards always have a header area that contains the wizard title, description, and a close button.
+1. **Header (modal wizard only):** Modal wizards always have a header area that minimally contains the wizard title and a close button. The title should be action oriented and reflect the intended outcome of the wizard, e.g. "Create resource." You may optionally also include a description below the wizard title.
 2. **Steps sidebar:** Enumerated steps are displayed in the sidebar. Steps can be fixed or updated as the user proceeds through the process (see Progressive Wizard).
 3. **Visited step:** A step that has been already visited. In most cases users can click this step to return to a prior point in the flow.
 4. **Current step:** The current step is always highlighted.
 5. **Disabled step:** Steps that follow the current step are usually disabled to enforce the sequential flow.
-6. **Body:** The contents of a step itself. Any valid form elements can be incorporated within the body. The size of the modal that contains the wizard can be adjusted based on the contents of the body section. If the height of the body section causes the bottom of the modal to push beyond the height of the viewport, a vertical scrollbar will appear. However it is recommended to break your workflow into small enough steps so that scrolling is not necessary on typical monitor sizes.
-7. **Button footer:** Buttons control the wizard flow. Default buttons are Next, Back, and Cancel. You may optionally add other actions, e.g. Skip to Finish or Start Over.
+6. **Step title:** The Step title is unique to each step, and should reinforce the outcome of the individual step. It may match the Current step indicated in the steps sidebar, or as in the example image, it may be more verbose.
```

Comment:
Are there step titles in the progressive wizard now? If not maybe we should state that.

Reply:
There should be step titles. Why wouldn't there be?

Reply:
Ya
Maybe I’m wrong I thought there wasn’t any the last time I looked

Reply:
@matthewcarleton Are you referring to what's being called a "Progressive Form?:" https://github.com/patternfly/patternfly-next/issues/1242  That still needs to be finalized and this documentation may need to be updated once we formally introduce that to the library.

Reply:
Ah of course. I was mixing these up. I’m thinking of the progressive form inside the wizard. Anyway we can revisit if needed when that gets introduced.